src/components/books/AddnewBook.js

```diff
@@ -18,6 +17,7 @@ function AddnewBook() {
   const onClick = () => {
     setHide(!hide);
   };
+  const addBook = () => { };
   const onSubmit = (e) => {
     const title = bookTitle.current.value;
     const enteredCategorie = categorie;
```

Comment:
- [ ] Kindly make sure your form to `Add` a new book is available and **ADDING** and **REMOVING** new books are possible as required for the project.